src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpRazorCommentsTest/RazorCommentBetweenCodeBlockAndMarkup.stree.txt

```diff
@@ -0,0 +1,41 @@
+RazorDocument - [0..39)::39 - [@{ }LF@* Hello World *@LF<div>Foo</div>]
+    MarkupBlock - [0..39)::39
+        MarkupTextLiteral - [0..0)::0 - [] - Gen<Markup> - SpanEditHandler;Accepts:Any
+            Marker;[];
+        CSharpCodeBlock - [0..4)::4
+            CSharpStatement - [0..4)::4
+                CSharpTransition - [0..1)::1 - Gen<None> - SpanEditHandler;Accepts:None
+                    Transition;[@];
+                CSharpStatementBody - [1..4)::3
+                    RazorMetaCode - [1..2)::1 - Gen<None> - SpanEditHandler;Accepts:None
+                        LeftBrace;[{];
+                    CSharpCodeBlock - [2..3)::1
+                        CSharpStatementLiteral - [2..3)::1 - [ ] - Gen<Stmt> - AutoCompleteEditHandler;Accepts:Any,AutoComplete:[<null>];AtEOL
+                            Whitespace;[ ];
+                    RazorMetaCode - [3..4)::1 - Gen<None> - SpanEditHandler;Accepts:None
+                        RightBrace;[}];
+        MarkupEphemeralTextLiteral - [4..6)::2 - [LF] - Gen<None> - SpanEditHandler;Accepts:Any
+            NewLine;[LF];
+        RazorComment - [6..23)::17
+            RazorCommentTransition;[@];
+            RazorCommentStar;[*];
+            RazorCommentLiteral;[ Hello World ];
+            RazorCommentStar;[*];
+            RazorCommentTransition;[@];
+        MarkupEphemeralTextLiteral - [23..25)::2 - [LF] - Gen<None> - SpanEditHandler;Accepts:Any
```

Comment:
Note: This was the reason the Mvc baselines had to be changed. Specifically the newline before this line https://github.com/aspnet/AspNetCore/blob/master/src/Mvc/test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/ProductListUsingTagHelpers.cshtml#L21 is no longer rendered.